docs/content/en/install/local/multi-cluster-kustomize/_index.md

```diff
@@ -192,16 +191,6 @@ Here is a summary of what the script does:
 Create a `ClientConfig` in the `kind-k8ssandra-0` cluster using the service account 
 token and CA cert from `kind-k8ssandra-1`:
```

Comment:
**Todo:** I think lines 182 to 192 should be removed. Right?

Reply:
I don't think so actually. k8ssandra-client is doing exactly the same thing, and I personally wouldn't run it unless I had some idea of what resources it was going to manipulate.

We probably shouldn't call it a script though, maybe line 182 does need to change.